package/gluon-l3roamd/files/etc/modules.d/gluon-l3roamd

```diff
@@ -0,0 +1 @@
+cfg80211
```

Comment:
Ugh, replace "mmfd" by "l3roamd" or vice-versa in the above message, whatever actually depends on *cfg80211* at the moment...